cmd/kops/update_cluster.go

```diff
@@ -249,18 +257,23 @@ func RunUpdateCluster(f *util.Factory, clusterName string, out io.Writer, c *Upd
 			fmt.Fprintf(sb, "Suggestions:\n")
 			fmt.Fprintf(sb, " * validate cluster: kops validate cluster\n")
 			fmt.Fprintf(sb, " * list nodes: kubectl get nodes --show-labels\n")
+			fmt.Fprintf(sb, " * download kubeconfig: kops export kubecfg --name %s\n", clusterName)
+			fmt.Fprintf(sb, " * read about installing addons: https://github.com/kubernetes/kops/blob/master/docs/addons.md\n")
```

Comment:
I worry we are recreating the docs.  Once this gets over a handful of items, nobody will read them